CMakeLists.txt

```diff
@@ -371,3 +129,12 @@ install(
   NAMESPACE "${namespace}"
   DESTINATION "${config_install_dir}"
 )
+
+option(BUILD_TESTING "Build unit test" ON)
```

Comment:
This option is built-in to CMake. No need to redeclare.

Reply:
Thanks. My last significant CMake experience was a *long* time ago.

Reply:
No worries :)